src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -259,7 +258,7 @@ void AttitudeEstimatorQ::task_main()
 
 	_sensors_sub = orb_subscribe(ORB_ID(sensor_combined));
 	_vision_sub = orb_subscribe(ORB_ID(vehicle_vision_attitude));
-	_mocap_sub = orb_subscribe(ORB_ID(att_pos_mocap));
```

Comment:
@bkueng Are we using this message in flightplot?

Reply:
https://github.com/PX4/flight_review/pull/111